src/error.rs

```diff
@@ -101,6 +101,42 @@ impl From<SpkiError> for Error {
     }
 }
 
+impl From<url::ParseError> for Error {
```

Comment:
I thought thiserror was supposed to handle these automatically, but they need impls now I guess.